src/AlertBox/AlertBox.tsx

```diff
@@ -62,9 +62,9 @@ const TYPES = {
 };
 
 AlertBox.propTypes = {
```

Comment:
yeah, planning on that, but it'll be a breaking change, so wanted to prototype a bit before merging there